diff --git a/BUILD.gn b/BUILD.gn
index c962350..37b880f 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -212,6 +212,9 @@ config("pdfium_common_config") {
 
     # May flag some issues when converting int to size_t.
     cflags += [ "-Wtautological-unsigned-zero-compare" ]
+
+    # Add the flag to prevent deprecated warnings from being errors
+    cflags_cc += [ "-Wno-error=deprecated-declarations" ]
   }
 
   if (!is_win && !is_clang) {
diff --git a/third_party/googletest/BUILD.gn b/third_party/googletest/BUILD.gn
index 2c46922..61f5504 100644
--- a/third_party/googletest/BUILD.gn
+++ b/third_party/googletest/BUILD.gn
@@ -35,6 +35,10 @@ config("gtest_config") {
   if (is_win) {
     cflags = [ "/wd4800" ]  # Unused variable warning.
   }
+
+  if (is_clang) {
+    cflags_cc = [ "-Wno-error=deprecated-declarations" ]
+  }
 }
 
 config("gmock_config") {
